### ISSUE TITLE
Bad classpath in graal-native-image Dockerfile

### ISSUE THREAD

**Issue by eginez** [micronaut-projects/micronaut-profiles#179]
Hi all

We recently spotted an incorrect rendering of the `@jarPath@` string in the Dockerfile template for graal-native-image
https://github.com/micronaut-projects/micronaut-profiles/blob/master/base/features/graal-native-image/skeleton/Dockerfile#L5

The @jarPath@ template is getting rendered to something like this: `builds/libs/file-*.jar`. Which will not properly expand to all the jars.

This is trivially fixed by just rendering the classpath like this: `'builds/libs/*'`. More info on  classpath wildcards: https://docs.oracle.com/javase/7/docs/technotes/tools/solaris/classpath.htm

**Comment by eginez** [comment]
With the current setup you will currently see this error:
```Error: Main entry point class 'build/libs/frontend-0.1.jar' not found.
Error: Use -H:+ReportExceptionStackTraces to print stacktrace of underlying exception
Error: Image build request failed with exit status 1
```

**Comment by ilopmar** [comment]
This is expanded now as `build/libs/xxxxxxx-*-all.jar`. It was fixed sometime ago and it works in Micronaut 1.3.1.
